plugins/doc_fragments/postgres.py

```diff
@@ -48,6 +48,10 @@ class ModuleDocFragment(object):
       - If the file exists, the server's certificate will be verified to be signed by one of these authorities.
     type: str
     aliases: [ ssl_rootcert ]
+  connect_params:
+    description:
+      - Any additional parameters to be passed to libpg as kwargs.
+    type: dict
```

Comment:
@Andersson007, doesn't this need a 'version added'? 2.3.0?

Reply:
@hunleyd yep, there should be `version_added: 2.3.0`, thanks for spotting

Reply:
I've added the version info.